CONTRIBUTING.md

```diff
@@ -31,6 +31,7 @@ into three categories:
 3. Comment on the issue saying you are going to work on it.
 4. Get familar with the developer guide relevant for you:
     * For C++ developers it is available here [DEVELOPER_GUIDE.md](wiki/cpp/DEVELOPER_GUIDE.md)
+    * For Python developers, cuML Python generally follows scikit-learn guidelines, but there are some additional considerations to ensure estimators work smoothly with all input types and configurable output types. See the documentation in `cuml.common.Base` for full details.
```

Comment:
Would be nice to include some guidelines for `input_to_cuml_array` in the [python developer_guide](https://github.com/rapidsai/cuml/blob/master/wiki/python/DEVELOPER_GUIDE.md). Maybe in a follow up PR.

Reply:
I just added a basic guidance to look to Base, but we should add more about input_utils in the future for sure.